### ISSUE TITLE
Robbing stage breaks game

### ISSUE THREAD

**Issue by DanielSaunders** [thyer/CS340winter2016#642]
This one will be tough to debug. After rolling, I got a 7. The robbing display popped up and disappeared almost immediately. It does not look like the robber changed positions. From here, I am unable to continue the game. However, I was able to close the client and join again and continue the game from the robbing map display.
![screenshot from 2016-03-25 22-26-15](https://cloud.githubusercontent.com/assets/13978907/14058052/d435dc5c-f2d8-11e5-8aaf-f3884857a3b4.png)

**Comment by misterpink14** [comment]
Could you post steps to replicate if you remember? Thanks

**Comment by DanielSaunders** [comment]
I don't remember doing anything especially unusual. I just clicked the roll button, then it told me I got a 7. At that point, when I clicked out of that dialog, it looked like the robbing display popped up and closed immediately. I don't remember if I clicked anything before it closed. But I couldn't really see the robbing display before it had closed anyway, so I'm not sure how I could have clicked anything else.
